website/content/en/preview/tasks/configuration.md

```diff
@@ -16,7 +16,7 @@ There are two main configuration mechanisms that can be used to configure Karpen
 |--|--|--|
 | AWS_DEFAULT_INSTANCE_PROFILE | \-\-aws-default-instance-profile | The default instance profile to use when provisioning nodes in AWS|
 | AWS_ENABLE_POD_ENI | \-\-aws-enable-pod-eni | If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource (default = false)|
-| AWS_ENI_LIMITED_POD_DENSITY | \-\-aws-eni-limited-pod-density | Indicates whether new nodes should use ENI-based pod density (default = true)|
+| AWS_ENI_LIMITED_POD_DENSITY | \-\-aws-eni-limited-pod-density | Indicates whether new nodes should use ENI-based pod density (default = true) DEPRECATED: Use `.spec.kubeletConfiguration.maxPods` to set pod density on a per-provisioner basis|
```

Comment:
The table in this doc is generated with the `make docgen` target. You'll need to set this in `pkg/utils/options/options.go`.

Reply:
Updated with the correct change